examples/test/videosphere/index.html

```diff
@@ -9,15 +9,41 @@
   <body>
     <a-scene stats>
       <a-assets>
-        <video id="lego" preload="true"
-               src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/t-197/walking_the_dog_360_video_short.webm"
+        <!-- Test anonymous video with source element. -->
+        <!-- Normally, one would give the video an id, e.g. id="lego" -->
```

Comment:
This comment's a bit confusing, without seeing the JavaScript `<script>` below. It seems like this example is being modified to test `<source>`. Could we just have a separate example that tests `<source>`s?